shared-libs/validation/test/validations.js

```diff
@@ -248,6 +248,61 @@ describe('validations', () => {
     });
   });
 
+  it.only('should fail uniquePhone if doc is found in db', () => {
+    clock = sinon.useFakeTimers();
+    sinon.stub(db.medic, 'query').resolves({
+      rows: [
+        {
+          id: 'original',
+          phone: '+9779841111111'
+        }
+      ]
+    });
+    const validations = [
+      {
+        property: 'phone_number',
+        rule: 'uniquePhone("phone_number")',
+        message: [
+          {
+            content: 'Duplicate phone',
+            locale: 'en',
+          },
+        ],
+      },
+    ];
+    const doc = {
+      _id: 'duplicate',
+      xyz: '+9779841111111',
+    };
+    return validation.validate(doc, validations).then(errors => {
+      assert.equal(errors.length, 1);
+    });
+  });
+
+  it.only('should pass uniquePhone if doc is not found in db', () => {
```

Comment:
I assume you meant to check phone and not doc here.

Reply:
Was checking if a contact doc has the same phone. Accepted suggestion to rename. Makes more sense.